tests/e2e/specs/activate-and-setup/setup-wizard.test.js

```diff
@@ -16,6 +16,8 @@ import {
 	verifyValueOfInputField
 } from '@woocommerce/e2e-utils';
 
+import { waitAndClick } from '../../env';
```

Comment:
@juliaamosova Imports in our packages cannot reference files in a different package using the relative path. This should be

```import { waitAndClick } from '@woocommerce/e2e-environment';

The reason the relative paths don't work is when another project uses the project the files in the relative paths outside the package don't exist.

Reply:
Thanks for cathing this @rrennick! Corrected.